pilot/pkg/xds/debug.go

```diff
@@ -669,6 +669,30 @@ func (s *DiscoveryServer) PushStatusHandler(w http.ResponseWriter, req *http.Req
 	_, _ = w.Write(out)
 }
 
+// PushContextDebug holds debug information for push context.
+type PushContextDebug struct {
```

Comment:
this might need to live in push context package if we want to access unexported fields in the future. fine for now though